:memo: Add gitihub actions documentation (#15)
kefranabg/bento-starter-docs: docs/setup/README.md

However we recommend you to go through optional steps to get a better developer experience :sunglasses:

> If you really don't want to us CI, you can disable it. Go to `Your project page → Settings → Actions` and select ̀`Disable Actions for this repository`.

Reviewer: ```suggestion
If you want to disable the CI, go to `Your project page → Settings → Actions` and select  ̀Disable Actions for this repository`.
```

1/PXcLCJ5BXAZ7ciFwkrrpikUbnMAMX8xRFmt16pLYudg9
```

Once you did that, you need to add this token in your Github project.

- Go to `Your project page → Settings → Secrets`
- Click on `Add a new secret`
- Fill `Name` input with `x`
- Copy the printed firebase token and paste into `Value` input
- Click on `Add secret button`

![Bento starter demo](/assets/img/secret-firebase.png)

All you need to do is to push your project code into `master` with git.

> - Back to a terminal run the following command :

Reviewer: ```suggestion
Back to a terminal, run the following command :
```
